test/mason/masonNewTest.good

```diff
@@ -1,2 +1,8 @@
+Created new application project: Test
+File structure: correct
+Created new application project: Test
+File structure: correct
 Created new library project: Test
 File structure: correct
+Created new light project: /Users/ben.mcdonald/chapel/test/mason
```

Comment:
looks like a local path made it in here

Reply:
Good catch. There were ~50 failed tests that came back from the paratest, so I'll work through those today and then re-push everything up, so this PR will get huge, but nothing is changing, just test updates, so I don't think another review will be necessary. Thanks!